README.md

```diff
@@ -17,7 +17,29 @@ A gem for easily using read replicas.
 
 ## Supported Versions
 
-Knockoff supports Rails 4, 5 and 6
+| Knockoff version: | 1.0 | 1.1.0 | 1.1.1 | 1.4 | 1.5 |
+| -------- | ------- | -------- | ------- | -------- |
```

Comment:
I think you need one more divider. There are six columns in total. Right now, it looks like this

<img width="910" alt="image" src="https://github.com/user-attachments/assets/cdc8fc3c-d7ab-404b-8e43-f5e5a8cb380d">